infra/build/functions/build_project.py

```diff
@@ -607,7 +608,7 @@ def run_build(oss_fuzz_project,
                              experiment=experiment)
 
 
-def get_args(description):
+def get_args(description, args):
```

Comment:
It seems weird passing args to a `get_args` function. And I saw below that you override the args with the return. Do you think we could just get the args with the function and then add the return of the function to the args?